docs/1.Welcome/What is Checkov.md

```diff
@@ -7,17 +7,17 @@ nav_order: 1
 
 # What is Checkov?
 
-Checkov is a static code analysis tool for scanning infrastructure as code (IaC) files for misconfigurations that may lead to security or compliance problems. Checkov includes more than 750 predefined policies to check for common misconfiguration issues. Checkov also supports the creation and contribution of of [custom policies](../3.Custom%20Policies/Custom%20Policies%20Overview.md).
+Checkov is a static code analysis tool for scanning infrastructure as code (IaC) files for misconfigurations that may lead to security or compliance problems. Checkov includes more than 750 predefined policies to check for common misconfiguration issues. Checkov also supports the creation and contribution of [custom policies](../3.Custom%20Policies/Custom%20Policies%20Overview.md).
 
 ## Supported IaC types
 
 Checkov scans these IaC file types:
 
-* Terraform (for AWS, GCP and Azure)
-* CloudFormation
-* Azure Resource Manager (ARM)
-* Kubernetes
-* Docker
+-   Terraform (for AWS, GCP and Azure)
```

Comment:
why is this change needed?

Reply:
I believe these changes came about from a code formatter that i have installed in my code editor. For the sake of consistency  i've reverted them back to their original bullets.